ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnecting-network/provision/cucushift-installer-rehearse-vsphere-upi-disconnecting-network-provision-chain.yaml

```diff
@@ -0,0 +1,26 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-disconnecting-network-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-vsphere-check
+  - chain: vsphere-provision-bastionhost
+  - ref: vsphere-provision-dnsmasq
```

Comment:
because dnsmasq is relying on basthost, suggest to move the step to `vsphere-provision-bastionhost-dnsmasq`